src/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -412,6 +424,12 @@ void CodeGen::genSSEIntrinsic(GenTreeHWIntrinsic* node)
             emit->emitIns_SIMD_R_R(INS_sqrtps, targetReg, op1Reg, TYP_SIMD16);
             break;
 
+        case NI_SSE_SqrtScalar:
+            assert(baseType == TYP_FLOAT);
+            assert(op2 == nullptr);
+            emit->emitIns_SIMD_R_R_R(INS_sqrtss, targetReg, op1Reg, op1Reg, TYP_SIMD16);
```

Comment:
@fiigii, on the original PR adding the API for these three intrinsics, you recommended having the API be:
`Vector128<float> SqrtScalar(Vector128<float> value)`, rather than 
`Vector128<float> SqrtScalar(Vector128<float> upper, Vector128<float> value)`.

However, this actually means we have to handle these intrinsics specially, since they have a different shape from the others. It also means the user has to inject other intrinsics to simulate the case where `upper` and `value` should not be the same (which most of the other scalar intrinsics already support in some form).

Is there an actual performance benefit to the user doing:
```
var input = Sse.MoveScalar(upper, value);
var result = Sse.SqrtScalar(input);
```

over just providing and allowing:
```
var result = SqrtScalar(upper, value);
```